src/components/stepper.js

```diff
@@ -63,43 +63,39 @@ const useStyles = makeStyles((theme) => ({
         width: 0,
         height: 0,
     },
-    [theme.breakpoints.down('xs') && theme.breakpoints.down('sm')]: {
-        downloadContainer: {
+    downloadContainer: (theme) => ({
+        [theme.breakpoints.down('sm')]: {
             minHeight: '120px',
         },
-        replaceIGMContainer: {
+    }),
+    replaceIGMContainer: (theme) => ({
+        [theme.breakpoints.down('sm')]: {
             minHeight: '120px',
         },
-    },
-}));
-
-const CustomStepper = withStyles({
-    root: {
-        padding: '30px 15px 25px 15px',
-    },
-})(Stepper);
-
-const CustomStepLabel = withStyles({
-    label: {
-        fontSize: '16px',
-    },
-})(StepLabel);
+    }),
+};
+const classes2 = {
+    stepperButtonContainer: (theme) => ({
+        ...classes.stepperGridMargins,
+        textAlign: 'center',
+        backgroundColor: theme.palette.background.paper,
+    }),
+};
```

Comment:
change "...classes.stepperGridMargins" to a const then move this className to main class object